src/foam/u2/wizard/DAOWizardlet.js

```diff
@@ -23,26 +27,17 @@ foam.CLASS({
       }
     },
     {
-      name: 'daoKey',
       class: 'String',
-      expression: function (of) {
-        return foam.String.daoize(of.name);
-      }
+      documentation: `
+        Optional path used to specify the location of the desired wizardlet\'s data object that will
```

Comment:
```suggestion
        Optional path used to specify the location of the desired wizardlet's data object that will
```

No need to escape the apostrophe when using backticks